src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/db/ProblemRepository.java

```diff
@@ -12,6 +12,15 @@
 @Repository
 public interface ProblemRepository extends JpaRepository<Problem, String> {
 
+    @Query("SELECT problem FROM Problem problem "
+        + "ORDER BY CASE severity "
+        + "         WHEN 'Critical'  THEN '0' "
+        + "         WHEN 'Urgent' THEN '1' "
+        + "         WHEN 'Warning'  THEN '2' "
+        + "         ELSE '2' "
```

Comment:
Why not `else '3'`? (just trying to understand ;) )
Does it mean that Warning and 'other' severity will be treated with the same sort priority?

Reply:
yeah exactly. When else would return more than 2 it would put new problems at the very end. I'm thinking that showing them early in the table would be more beneficial.

Reply:
@zielinskikamil that's what I meant by using the case when in the query.